.github/workflows/deploy.yml

```diff
@@ -0,0 +1,21 @@
+name: Build and Deploy
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install Dependencies
+        run: npm install -g firebase-tools
+      - name: Checkout Repo
+        uses: actions/checkout@master
+      - name: Install Dependencies
+        run: npm install
+      - name: Build & Deploy
+        run: npx nx deploy
```

Comment:
`ng deploy` should be enough here

Reply:
Doesn't work - https://github.com/attrib/Lostark-helper/actions/runs/3130437161/jobs/5080683736